src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,105 @@
+/* eslint-disable no-alert */
 import { useState } from 'react';
+import { Movie } from '../../types/Movie';
 import { TextField } from '../TextField';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+type Props = {
+  onAdd: (movie: Movie) => void;
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
```

Comment:
Try to make more meaningful component names
```suggestion
export const AddMovieForm: React.FC<Props> = ({ onAdd }) => {
```